metrics/web/test/project_groups/presentation/state/project_groups_notifier_test.dart

```diff
@@ -0,0 +1,874 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:metrics/common/domain/entities/persistent_store_error_code.dart';
+import 'package:metrics/common/domain/entities/persistent_store_exception.dart';
+import 'package:metrics/common/presentation/models/persistent_store_error_message.dart';
+import 'package:metrics/common/presentation/models/project_model.dart';
+import 'package:metrics/common/presentation/strings/common_strings.dart';
+import 'package:metrics/project_groups/domain/entities/project_group.dart';
+import 'package:metrics/project_groups/domain/usecases/add_project_group_usecase.dart';
+import 'package:metrics/project_groups/domain/usecases/delete_project_group_usecase.dart';
+import 'package:metrics/project_groups/domain/usecases/parameters/add_project_group_param.dart';
+import 'package:metrics/project_groups/domain/usecases/parameters/delete_project_group_param.dart';
+import 'package:metrics/project_groups/domain/usecases/parameters/update_project_group_param.dart';
+import 'package:metrics/project_groups/domain/usecases/receive_project_group_updates.dart';
+import 'package:metrics/project_groups/domain/usecases/update_project_group_usecase.dart';
+import 'package:metrics/project_groups/presentation/state/project_groups_notifier.dart';
+import 'package:metrics/project_groups/presentation/view_models/project_checkbox_view_model.dart';
+import 'package:metrics/project_groups/presentation/view_models/project_group_card_view_model.dart';
+import 'package:metrics/project_groups/presentation/view_models/project_group_dialog_view_model.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../../test_utils/matcher_util.dart';
+
+void main() {
+  group("ProjectGroupsNotifier", () {
+    const projectGroupId = 'projectGroupId';
+    const projectGroupName = 'projectGroupName';
+
+    final projectGroup = ProjectGroup(
+      id: projectGroupId,
+      name: projectGroupName,
+      projectIds: const [],
+    );
+
+    const project = ProjectModel(id: 'projectId', name: 'projectName');
+
+    const projects = [
+      project,
+      ProjectModel(id: '1', name: 'name1'),
+      ProjectModel(id: '2', name: 'name2'),
+    ];
+
+    final projectGroups = [
+      projectGroup,
+      ProjectGroup(id: '1', name: 'name1', projectIds: const []),
+      ProjectGroup(id: '2', name: 'name2', projectIds: const []),
+    ];
+
+    final addProjectGroupUseCase = AddProjectGroupUseCaseMock();
+    final updateProjectGroupUseCase = UpdateProjectGroupUseCaseMock();
+    final deleteProjectGroupUseCase = DeleteProjectGroupUseCaseMock();
+    final receiveProjectGroupUpdates = ReceiveProjectGroupUpdatesMock();
+
+    final projectGroupsNotifier = ProjectGroupsNotifier(
+      receiveProjectGroupUpdates,
+      addProjectGroupUseCase,
+      updateProjectGroupUseCase,
+      deleteProjectGroupUseCase,
+    );
+
+    setUpAll(() async {
```

Comment:
It looks like the count of selected projects is not updated on UI when editing group:
See that only 1 project select but the count is 6:

![image](https://user-images.githubusercontent.com/442086/86624321-cace3780-bfcb-11ea-9daa-34e7d856bbea.png)

I think it worked before - or maybe there is some issue with my build - let me know.